### DISCUSSION TITLE
Update to Simplified Chinese locale [zh]

### DISCUSSION THREAD

**Discussion by archonwang** [aclap-dev/video-downloadhelper#365] [Language & Translation]
[messages.json](https://github.com/aclap-dev/video-downloadhelper/files/14846322/messages.json)
I translated some words for  V8.2.2.8 (stable) on  Firefox. If you have time, plz Check it. Thanks.

**Comment by paulrouget** [comment]
Thank you! I will add that to the upcoming version.
